projects/ngrx.io/content/guide/store/why.md

```diff
@@ -1,49 +1,49 @@
 # Why use NgRx Store for State Management?
 
-NgRx Store provides state management for creating maintainable explicit applications, by single state and the use of actions in order to express state changes. In cases where you don't need a global application-wide solution to manage state, consider using [NgRx ComponentStore](guide/component) which provides a solution for local state management.
+NgRx Store provides state management for creating maintainable, explicit applications via single state and the use of actions in order to express state changes. In cases where you don't need a global, application-wide solution to manage state, consider using [NgRx ComponentStore](guide/component) which provides a solution for local state management.
```

Comment:
I'd like to use some other word than `via`, as it's not a universal term

Reply:
I've reworked the wording in this section to hopefully make it a bit better..

Reply:
Hi Brandon, should I mark this as resolved, or does the reviewer typically do it?